cesium_app/util.py

```diff
@@ -35,3 +36,38 @@ def robust_literal_eval(val):
 def secure_filename(filename):
     filename = filename.encode('utf-8')
     return hashlib.sha256(filename).hexdigest()[:20]
+
+
+def prediction_to_csv(pred, outpath=None):
```

Comment:
Docstring missing, and this function looks quite complex.  Any easy ways to simplify it?